### ISSUE TITLE
Quick-start tutorial for using VideoLab

### ISSUE THREAD

**Issue by jwmueller** [cleanlab/cleanvision#239]
Goal: create a tutorial notebook that looks like this one: https://cleanvision.readthedocs.io/en/latest/tutorials/tutorial.html

But runs VideoLab instead on an example video dataset. 
The tutorial should be a notebook PR'd to here: 
https://github.com/cleanlab/cleanvision/tree/main/docs/source/tutorials
(this new notebook should look just like the other tutorials)

The example dataset should have at least one instance of each type of issue that VideoLab can detect, but should not be large (so that user's can fetch it and run videolab quickly).  Please provide a link that shows others have the right to distribute this dataset. 

The first step of the notebook should be to unzip a zipped folder containing all the videos.
You can share this folder with our team who will make it available in S3 and update the tutorial as needed to load it from there.

**Comment by lafemmephile** [comment]
So far I have only been able to to find the *example video data* for the following issues:
```python
['blurry',
 'dark',
 'exact_duplicates',
 'light',
 'near_duplicates',
 'low_information']
```
This was accomplished using this [notebook](https://colab.research.google.com/github/lafemmephile/videolab/blob/master/notebooks/video_test_data_issues.ipynb) which uses `Videolab` to find issues in the [WebVid-10M dataset](https://maxbain.com/webvid-dataset/). The issues are written to a *JSON* file (which can be found [here](https://github.com/lafemmephile/videolab/blob/master/notebooks/webvid_issues.json)), but are missing the following issues:
```python
['grayscale',
 'odd_size',
 'odd_aspect_ratio']
```

Should the *example video dataset* contain videos for the above *missing* issues? And if so do you have any suggestions/insight into finding them? I guess I could just create them myself ... that might be the easiest solution.

**Comment by jwmueller** [comment]
Yep this looks great so far!  It would be super helpful if you're also able to add in just one short video with each of these issues:

> ```python
> ['grayscale',
>  'odd_size',
>  'odd_aspect_ratio']
> ```
 

As you mentioned it might be easiest to create videos that have these issues yourself, or you can just link to short clips that exhibit these properties you found online.

**Comment by lafemmephile** [comment]
I have built, yet another, [Jupyter Notebook](https://colab.research.google.com/github/lafemmephile/videolab/blob/master/notebooks/building_videolab_example_dataset.ipynb) that actually takes the `Imagelab` example dataset (as referenced above) and *compiles* videos out of all the images that have issues.

On the surface, as you might expect, this works, except for two issues:
+ `grayscale`
+ `odd_size`

No matter what I do, these do not want to work. Trust me when I say I have tried *EVERYTHING*: downloading videos, manually editing videos, etc ... I suspect the issue lies in how `grayscale` and `odd_size` are scored, or how they are being *aggregated* in `Videolab` ...

**Comment by lafemmephile** [comment]
@jwmueller Here is the attached *zip file* of the videos compiled in the aforementioned [notebook](https://colab.research.google.com/github/lafemmephile/videolab/blob/master/notebooks/building_videolab_example_dataset.ipynb):

[videolab_example_dataset.zip](https://github.com/cleanlab/cleanvision/files/13405026/videolab_example_dataset.zip)

**Comment by lafemmephile** [comment]
@jwmueller You now have EVERYTHING I have, including links to the *notebooks* that created the attached *zip file*. Again, the attached zip file has neither `grayscale` nor `odd_size` issues. But it is a start :+1:.
